packages/core/src/RPA/core/windows/window.py

```diff
@@ -76,14 +71,18 @@ def list_windows(
                 fullpath = win32process.GetModuleFileNameEx(handle, 0)
```

Comment:
Above we use:
```python
handle = win32api.OpenProcess(win32con.PROCESS_ALL_ACCESS, False, pid)
```
but in original windows library we [have](https://github.com/robocorp/rpaframework/blob/2dc3bf93b115f4c8816d48aa2b5b8b8a2c9c79c5/packages/windows/src/RPA/Windows/keywords/window.py#L284-L286):
```python
handle = win32api.OpenProcess(win32con.PROCESS_QUERY_LIMITED_INFORMATION, False, pid)
```
and I suspect this is because we require full info/privileges in core compared to windows which limits to minimum required access.

The question is: Do we keep separate access levels in these two (like it is right now) by defaulting to `PROCESS_ALL_ACCESS` if the windows library doesn't specify a different level? If not, shall we use `PROCESS_ALL_ACCESS` in both?

Reply:
I changed this in the Windows library because of principle of use least privilege needed, the `PROCESS_QUERY_LIMITED_INFORMATION` is enough to get window handle.

Reply:
So `PROCESS_QUERY_LIMITED_INFORMATION` would be just enough for both, thx!